clients/linux/src/app.rs

```diff
@@ -80,11 +80,16 @@ impl LbApp {
             messenger: m,
         };
 
+        let background_work = lb_app.state.borrow().background_work.clone();
+
+        thread::spawn(move || BackgroundWork::init_background_work(background_work));
+
         if s.borrow().auto_save {
```

Comment:
We check to see if we should send true.

We should just send the value of the boolean regardless of true or false.